build.gradle

```diff
@@ -6,7 +6,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:4.0.0'
+        classpath 'com.android.tools.build:gradle:4.0.1'
```

Comment:
Please do not update build tools when committing a thumbnail. This should go into a separate PR.